cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,27 @@
+/*Login and access profile test + form validation test*/
+
+describe('Login Tests', () => {
+  const validEmail = 'test123@stud.noroff.no';
+  const validPassword = 'test1234';
+  const invalidEmail = 'test@hotmail.com';
+  const invalidPassword = 'test';
+
+  it('allows a user with correct credentials to log in', () => {
+    cy.visit('/');
+    cy.get('#registerModal').contains('Login').click();
+    cy.get('#loginForm').should('be.visible');
+    cy.get('#loginEmail').type(validEmail);
+    cy.get('#loginPassword').type(validPassword);
+    cy.get('button[type=submit]').contains('Login').click();
+    cy.wait(5000);
```

Comment:
There is a missing step to check that this succeeded. You could check localStorage for example